tools/generate_uuids.py

```diff
@@ -4,10 +4,12 @@
 
 parser = argparse.ArgumentParser(description="Generate UUIDs for info*.json files that don't already have them.")
 parser.add_argument("directory", help="the directory to search for info*.json files")
+parser.add_argument("-v", "--verbose", action="store_true", help="List all the files for which uuid's were generated")
 parser.add_argument("-n", "--new", action="store_true", help="generate new UUIDs for all files, even if they already have one")
 args = parser.parse_args()
 
-skip_dirs = [".git"]
+# Added elements in the skip directory because elements should not contain a uuid
+skip_dirs = [".git","elements"]
```

Comment:
Follow PEP8 style guidelines: https://www.python.org/dev/peps/pep-0008/

In this case, space after comma.